builtin/async/init.lua

```diff
@@ -1,4 +1,20 @@
 
+if core.get_cache_path == nil then -- TODO
+
+core.log("info", "Initializing Asynchronous environment for game")
+
+function core.job_processor(func, serialized_params)
+	local params = core.deserialize(serialized_params)
```

Comment:
Same for the params. As long as you store the varargs in a list, it will be `nil`-terminated. Either note this in the docs or fix it. See: http://lua-users.org/wiki/VarargTheSecondClassCitizen

Reply:
> `minetest.after` has the same issue.

...in this case I'd prefer just documenting it.